auth/client/src/main/java/org/wildfly/security/auth/client/AuthenticationContext.java

```diff
@@ -102,10 +102,6 @@ public static AuthenticationContext captureCurrent() {
         return node == null ? otherNode : otherNode == null ? node : node.withAll(otherNode);
     }
 
-    private static <T> RuleNode<T> withAll(RuleNode<T> node, RuleNode<T> otherNode, int idx) {
```

Comment:
@boris-unckel Just a minor, I would maybe not delete this method. It could be used later as it is generic enough. Otherwise LGTM

Reply:
@Skyllarr thanks for your feedback. I have removed the file from the PR. Please check again.

Reply:
@boris-unckel Thank you.